app/src/main/java/net/skyscanner/backpack/demo/meta/Component.kt

```diff
@@ -25,4 +25,5 @@ import javax.annotation.concurrent.Immutable
 @Retention(AnnotationRetention.BINARY)
 annotation class Component(
   val name: String,
+  val isToken: Boolean = false,
```

Comment:
To have tokens section

Reply:
can you cover this with tests/test stories?